### PR TITLE
UI requirement edit

### PR DESCRIPTION
**Pull request by jolivo07**

### CODE REVIEW

**Review by mperezlamadrid — APPROVED** [review]

File: app/templates/requirement/form.plush.html [hunk 1]
```diff
@@ -1,10 +1,26 @@
...
-        <button class="btn btn-success mt-3" role="submit">Save</button>
-        
\ No newline at end of file
+<div class="d-flex justify-content-between mb-4">
+        <div class="w-48">
+            <%= f.InputTag("Title") %>
+        </div>
+        <div class="w-48" >
+            <%= f.SelectTag("ServiceDepartmentID", {options: departments , label: "Service Department",class:"form-select" }) %>
+        </div>
```
Comment (rdiazrovira) [inline app/templates/requirement/form.plush.html]: You can use "grid" to get the correct distribution for those controls. Check the Bootstrap documentation (Grid system).

File: app/assets/css/requirement.scss [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+}
+.size-box{
+    width: 100%;
+}
+.resize-none{
+    resize: none !important;
+  }
+.btn-save-requirement:hover{
+    background-color: #00AD9B;
+}
```
Comment (rdiazrovira) [inline app/assets/css/requirement.scss]: Can you extract that color and move it to a css constant ?

File: app/assets/css/requirement.scss [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+.h-115px{
+    height: 115px;
+}
+.bg-green-ligth{
+    background-color: #00AD9B;
+}
+.size-box{
+    width: 100%;
+}
```
Comment (rdiazrovira) [inline app/assets/css/requirement.scss]: Bootstrap already has a class for it.

**Review by rdiazrovira — CHANGES_REQUESTED** [review]
Please take a look at the comments.

**Review by rdiazrovira — APPROVED** [review]

**Pull request merged**